examples/Console/TestPrometheusExporter.cs

```diff
@@ -16,80 +16,63 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Diagnostics.Metrics;
 using System.Threading;
 using System.Threading.Tasks;
 using OpenTelemetry;
 using OpenTelemetry.Metrics;
 using OpenTelemetry.Trace;
 
-namespace Examples.Console
-{
-    internal class TestPrometheusExporter
-    {
-        private static readonly Meter MyMeter = new Meter("TestMeter");
-        private static readonly Counter<double> Counter = MyMeter.CreateCounter<double>("myCounter", description: "A counter for demonstration purpose.");
-        private static readonly Histogram<long> MyHistogram = MyMeter.CreateHistogram<long>("myHistogram");
-        private static readonly ThreadLocal<Random> ThreadLocalRandom = new ThreadLocal<Random>(() => new Random());
-
-        internal static object Run(int port, int totalDurationInMins)
-        {
-            /*
```

Comment:
i'd still keep a known prometheus.yml for quickly testing.

We include it in almost all examples in this folder
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/Console/TestOtlpExporter.cs#L30

Reply:
I've added a simple yml file.

Curious, do we treat this project as an "example", a "test" or both?

Reply:
yes its a mix of both now. Agree we need a readme with instructions on how to run etc.

(I use them a lot, (and I believe other contributors as well), to quickly test things out.)